libcxx/src/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set(LIBCXX_SOURCES
   include/ryu/ryu.h
   include/to_chars_floating_point.h
   include/from_chars_floating_point.h
+  log_hardening_failure.cpp
```

Comment:
I think this should go under `LIBCXX_EXPERIMENTAL_SOURCES` and in `experimental/log_hardening_failure.cpp`.